slave/build.py

```diff
@@ -262,6 +262,16 @@ def make(self):
             args = [os.path.join('.', 'mach'), 'build' ,'--release']
             Run(args, self.env.get())
 
+    def build(self, puller):
+        # Call parent's build
+        super(ServoBuilder, self).build(puller)
+        # Read info.json file back and modify only shell=False.
+        with open(os.path.join(self.folder, "info.json"),  'r+') as info_file:
+            info = json.load(info_file)
+            info["shell"] = False
+            info_file.seek(0)
+            json.dump(info, info_file)
```

Comment:
This seems like a bit of a hack. Instead of doing that:
Can you put in the constructor of Builder:
self.shell = True

And in the Builder build method read that value
to write into the json file, instead of "True".

And in the constructor of ServoBuilder after calling the parent constructor,
put self.shell to False.